pkg/translator/testdata/simple_grpc_collector_config.yaml

```diff
@@ -6,11 +6,13 @@ receivers:
             http:
                 endpoint: ${STRAWS_COLLECTOR_POD_IP}:1234
 exporters:
+    debug/ShowMeTheOlly:
```

Comment:
I haven't tested yet in a real collector whether the debug exporter is happy with being given a name.

Reply:
it can have a name.